### ISSUE TITLE
Brew Formula? :)

### ISSUE THREAD

**Issue by armenr** [Abdenasser/neohtop#16]
Are you planning on putting together a homebrew formula for this? :)

**Comment by laciferin2024** [comment]
Stats is only 6.6 mb, but has a homebrew formula

**Comment by aqeelat** [comment]
I agree that a brew formula is much needed. It also gives users the confidence that the software is trustworthy

**Comment by pascaliske** [comment]
@Abdenasser I can create a PR later to add this if you are willing to accept it. 🙂

**Comment by Abdenasser** [comment]
> I agree that a brew formula is much needed. It also gives users the confidence that the software is trustworthy

don't really think it's more secure with a brew formula VS an apple Notarized app (apple now move apps that are not signed right away to trash)

**Comment by laciferin2024** [comment]
Yeah that's true, it doesn't matter as long as u have self updating application. With brew , applications can be updated on demand

**Comment by Abdenasser** [comment]
NeoHtops dmgs are all signed with my apple developer account (and all versions should be, otherwise the trash bins are waiting).. I figured that out the hard way after I tried sharing the app with my wife and another friend.. they were both saying it didn't work and it didn't even say the usual "this app is downloaded from internet do you want to open it", it did just had a "move to trash button"

> Important
Starting November 1, 2023, the Apple notary service no longer accepts uploads from altool or Xcode 13 or earlier. If you notarize your Mac software with the Apple notary service using the altool command-line utility or Xcode 13 or earlier, you need to transition to the notarytool command-line utility or upgrade to Xcode 14 or later.

reference:https://developer.apple.com/documentation/security/notarizing-macos-software-before-distribution
they just started doing this, this month.

**Comment by max-horvath** [comment]
Hi there,

I noticed that the request to add neohtop to Homebrew was closed, but I'd like to ask you to please reconsider.

Adding neohtop to Homebrew would be a great opportunity to expand the app's reach and make it easily accessible to a large community of macOS users. While you might not personally use Homebrew, many macOS users rely on it as their primary method for installing software, and having neohtop available there could introduce it to thousands of new users.

Just to highlight the impact: in the last 30 days alone, over 1.2 million macOS applications [were installed](https://formulae.brew.sh/analytics/cask-install/30d/?t) via Homebrew casks. With neohtop available on Homebrew, you could easily reach thousands of new users who might not otherwise discover the app.

I understand that signing neohtop with your Apple Developer account is a requirement for the distribution outside of the Apple Appstore, but this aligns with [Homebrew's guidelines](https://github.com/Homebrew/brew/pull/17002) for smooth and secure installation. If you're open to it, I'd be happy to help in any way I can to support the process.

**Comment by laciferin2024** [comment]
> . I figured that out the hard way after I tried sharing the app with my wife and another friend.. they were both saying it didn't work and it didn't even say the usual "this app is downloaded from internet do you want to open it", it did just had a "move to trash button"

@Abdenasser that's correct. Its harder to run internet downloaded apps from catalina onwards.But its not imposible , u can still run em. 


Notarized apps are definetely easier to install for non-developers but devs tend to cling on to command line . 'for em , `brew install` and `brew upgrade` is the way to go. That's probably why a ton of desktop applications are published to brew.

**Comment by laciferin2024** [comment]
@max-horvath I believe it might be a bit early to add the complexity in a manual deployment pipeline for the brew formula.
